go/tools/builders/cgo2.go

```diff
@@ -196,7 +197,8 @@ func cgo2(goenv *env, goSrcs, cgoSrcs, cSrcs, cxxSrcs, objcSrcs, objcxxSrcs, sSr
 	mainBin := filepath.Join(workDir, "_cgo_.o") // .o is a lie; it's an executable
 	args = append([]string{cc, "-o", mainBin, mainObj}, cObjs...)
 	args = append(args, combinedLdFlags...)
-	if err := goenv.runCommand(args); err != nil {
+	originalErrBuf := &bytes.Buffer{}
```

Comment:
nit: prefer `var originalErrBuf bytes.Buffer` ([style guide](https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs)) and use `&originalErrBuf` as needed. It also more explicit when you need a pointer instead of a value.